test/templates/config.ru

```diff
@@ -0,0 +1,90 @@
+require "action_view"
+require "pathname"
+
+TEST_ROOT = Pathname(__FILE__).dirname
+
+class FakeMiddleware
+  DEFAULT_HEADERS = {"Content-Type" => "application/javascript"}
+
+  def initialize(opts)
+    @headers        = opts.fetch(:headers, DEFAULT_HEADERS)
+    @req_path_regex = opts[:req_path_regex]
+    @view_path      = opts[:view_path]
+  end
+
+  def call(env)
+    [200, headers, [render(req_path(env))]]
+  end
+
+  private
+
+  attr_reader :headers
+  attr_reader :req_path_regex
+  attr_reader :view_path
+
+  # extract target path from REQUEST_PATH
+  def req_path(env)
+    env["REQUEST_PATH"].match(req_path_regex)[1]
+  end
+
+  def render(template)
+    view.render(template: template, layout: nil)
+  end
+
+  def view
+    @view ||= create_view
+  end
+
+  def create_view
+    lookup_context = ActionView::LookupContext.new(view_path)
+    lookup_context.cache = false
+    FakeView.new(lookup_context)
+  end
+
+  class FakeView < ActionView::Base
+    def render_inlined_string(template)
+      render(template: template, layout: "layouts/inlined_string")
+    end
+  end
+end
+
+module Helper
+  def self.relative_path(path)
+    TEST_ROOT.join path
+  end
+  def self.root_path(path)
```

Comment:
You just use this once, you can drop the method and the `Helper` module as a whole.